### PR TITLE
Ask for example Repository in bug_report.md

### PR DESCRIPTION
**Pull request by JesterOrNot**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -14,4 +14,7 @@ about: Create a report to help us improve
 <!-- A clear and concise description of what you expected to happen. -->
 
 **Additional Information**
-<!-- For instance, relevant logs from terminals or the browser's devtool console. -->
+<!-- For instance, relevant logs from terminals or the browser's Dev-Tools console. -->
```
Comment (jankeromnes) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
<!-- For instance, relevant logs from terminals or the browser's DevTools console. -->
```

**Comment by JesterOrNot** [comment]
But feature request has it's own template

**Comment by jankeromnes** [comment]
Also, it feels like a repository URL could already be part of "Additional Information".

I'd like to keep the number of questions small in this issue template. Do you think it could be feasible to merge the "Additional Information" and the "Example Repository" sections?

**Comment by JesterOrNot** [comment]
Yea, good idea!

**Comment by jankeromnes** [comment]
> But feature request has it's own template

That's an excellent point. I withdraw my feature request counter-argument. 🙂

**Pull request merged**